show-vars.md

```diff
@@ -0,0 +1,121 @@
+---
+title: SHOW (session settings)
+summary: The SHOW statement displays the current settings for the client session.
+toc: false
+---
+
+The `SHOW` [statement](sql-statements.html) can display the value of either one or all of
+the session setting variables. Some of these can also be configured via [`SET`](set-vars.html).
+
+<div id="toc"></div>
+
+## Required Privileges
+
+No [privileges](privileges.html) are required to display the session settings.
+
+## Synopsis
+
+{% include sql/diagrams/show_var.html %}
+{% include sql/diagrams/show_all.html %}
```

Comment:
Just as with the diagrams on `set_var.md`, an empty line needs to be inserted between these two includes. 

Also, I think you need to regenerate the diagrams themselves one more time:

`cd generate`
`go run main.go --filter show_var`
`go run main.go --filter show_all`